lib/node_modules/@stdlib/stats/base/svarianceyc/src/main.c

```diff
@@ -78,8 +78,8 @@ float API_SUFFIX(stdlib_strided_svarianceyc_ndarray)( const CBLAS_INT N, const f
 		di = (double)i;
 		v = X[ ix ];
 		sum += v;
-		d = (float)(di*(double)v) - sum;
-		S += (float)(1.0/(di*(di-1.0))) * d * d;
+		d = (float)( di * (double)v ) - sum;
+		S += (float)( 1.0 / ( di * ( di-1.0 ) ) ) * d * d;
```

Comment:
we don't need this change, this can be reverted back
```suggestion
		d = (float)(di*(double)v) - sum;
		S += (float)(1.0/(di*(di-1.0))) * d * d;
```